### DISCUSSION TITLE
which OT features are enabled by default?

### DISCUSSION THREAD

**Discussion by pr-apes** [speedata/publisher#383] [Q&A] [answered]
@pgundlach,

I’m afraid I cannot find which _OpenType_ features are enabled by default with _Publisher_ (with `mode="harfbuzz"`, if it is relevant).

I thought `+liga,+kern` were enabled, only to discover that the first one is not.

Many thanks for your help.

**Comment by pgundlach** [comment, marked as answer]
kern = true, liga = false is the default, I'll put this into the documentation.

**Reply by pgundlach** [reply]
I've opened an issue (#384). Thanks for the report.

**Reply by pr-apes** [reply]
Many thanks for your reply.

**Comment by pgundlach** [comment]
There is also https://harfbuzz.github.io/shaping-opentype-features.html which explains the default features of harfbuzz. I only deactivate liga.
